ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh

```diff
@@ -94,9 +94,16 @@ LIST_OF_KUBEOBJECTS=$(find "${CUSTOM_KUBEOBJECTS_PATH}"* -type f -name kube_obje
 if [ -n "${LIST_OF_KUBEOBJECTS[0]}" ] ; then
     for i in "${LIST_OF_KUBEOBJECTS[@]}"
     do
-        echo "The following custom kubeobject has been found ! $i"
-        echo "Applying kube_objects on to the Namespace $OO_INSTALL_NAMESPACE"
-        oc apply -f "$i" -n "$OO_INSTALL_NAMESPACE"
+	echo "The following custom kubeobject has been found ! $i"
+	echo "Check if the namespace is mentioned inside the kubeobjects"
+        if grep -inq "namespace" $i
```

Comment:
what is a benefit of using parameter `-n` in quiet mode `-q` ?
```
       -n, --line-number
              Prefix  each  line of output with the 1-based line number within
              its input file.
```

Reply:
It should be `grep -iq` since the `-n` prints the line number, but `-q` suppresses the output.